.github/workflows/dashboards-observability-test-and-build-workflow.yml

```diff
@@ -51,11 +51,10 @@ jobs:
           cd OpenSearch-Dashboards/plugins/dashboards-observability
           yarn osd bootstrap
 
-      # TODO enable unit tests when ready
-      - name: Test Panels
+      - name: Test all dashboards-observability modules
         run: |
           cd OpenSearch-Dashboards/plugins/dashboards-observability
-          yarn test ./public/components/custom_panels/ --coverage
+          yarn test --coverage
```

Comment:
Just curious- any specific reason we are including the coverage in the workflow here? Just for more transparency or will the CI break if we hit below a certain percentage?

Reply:
Currently it is just for more transparency. In future if we see a lot of regressions, then we can add the minimum percentage criteria here.